src/ReflectionFileNamespace.php

```diff
@@ -83,6 +83,9 @@ class ReflectionFileNamespace
      */
     public function __construct($fileName, $namespaceName, Namespace_ $namespaceNode = null)
     {
+        if (!is_string($fileName)) {
```

Comment:
The only reason I made this addition was that it was flagged by Scrutinizer:

> Documentation Bug introduced 10 months ago by @lisachenko
> It seems like `$fileName` can also be of type `array` or `Boolean`. However, the property `$fileName` is declared as type `string`. Maybe add an additional type check?

(Same comment as above)